doc/source/reference/arrays.ndarray.rst

```diff
@@ -597,19 +595,17 @@ Container customization: (see :ref:`Indexing <arrays.indexing>`)
    ndarray.__setitem__
    ndarray.__contains__
 
-Conversion; the operations :func:`complex()`, :func:`int()`,
-:func:`long()`, :func:`float()`, :func:`oct()`, and
-:func:`hex()`. They work only on arrays that have one element in them
+Conversion; the operations :func:`complex()`, :func:`int()` and
+:func:`float()`.
```

Comment:
Nit: Putting `complex()` last would make it match the order of the list below

Reply:
fixed